node/muneem/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "muneem": "~2.4.3"
```

Comment:
could use a strict dependency constraint, https://renovatebot.com/ will take care of update ?